### PR TITLE
Newfeature

### PR DESCRIPTION
**Pull request by jpriest-dev**
@james-priest I've updated the REAME.md file. What do you think?

### CODE REVIEW

**Comment by james-priest** [comment]
So far so good. @jpriest-dev Can you please include a scripts.js file?

**Comment by jpriest-dev** [comment]
Ok its up there

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var Programmer = {
+    name: "James Priest",
+    email: "james-priest@outlook.com",
+    // projects: [{
```
Comment (james-priest) [inline js/script.js]: Arbitrary comment... @jpriest-dev Can we remove the commented out stuff?

**Review by james-priest — APPROVED** [review]

**Comment by james-priest** [comment]
@jpriest-dev this is closed out

**Pull request merged**